### PR TITLE
PostAndCommentList.jsx: Pass `site` to `<Ad />` instead of `srnames`.

### PR DESCRIPTION
**Pull request by dwick**
:eyeglasses: @schwers 

Did this for `ListingList`, but missed this component.

### CODE REVIEW

File: src/views/components/listings/PostAndCommentList.jsx [hunk 1]
```diff
@@ -58,10 +62,17 @@ export default class PostAndCommentList extends BaseComponent {
...
-  computeSubredditNames() {
-    return uniq(this.props.postsAndComments.map(function(l) {
-      return l.subreddit;
-    }));
+  getSite() {
+    let site = _FRONTPAGE_NAME;
+
+    if (this.props.multi) {
+      site = `/user/${this.props.multiUser}/m/${this.props.multi}`
+    } else if (this.props.subredditName &&
```
Comment (schwers) [inline src/views/components/listings/PostAndCommentList.jsx]: Can this use `lib/isFakeSubreddit` instead?

**Comment by schwers** [comment]
:nail_care: Looks good, just one small question

**Comment by dwick** [comment]
:haircut:

**Comment by schwers** [comment]
:fish:

**Pull request merged**